### PR TITLE
chore: add swc-mantainers to CODEOWNERS

### PR DESCRIPTION
**Pull request by rubencarvalho**
## Related issue(s)

- SWC-190

## Motivation and context

All pull requests should automatically assign reviewers from the @adobe/swc-maintainers team for all files in the repository and all members of the team should receive notifications for review requests.

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [x] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [x] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

**Review by TarunAdobe — APPROVED** [review]
LGTM! On a side note Im curious about how to test these changes before merging

**Comment by rubencarvalho** [comment]
> LGTM! On a side note Im curious about how to test these changes before merging

Unfortunately the changes to the CODEOWNERS will only take effect for subsequent PRs. I will quickly test it after merging this.

**Pull request merged**
